components/stockchart/labels-template-and-format.md

```diff
@@ -0,0 +1,242 @@
+        </StockChartValueAxis>
+    </StockChartValueAxes>
+
+    <StockChartSeriesItems>
+        <StockChartSeries Type="StockChartSeriesType.Candlestick"
+                            Name="Product 1"
+                            Data="@StockChartProduct1Data"
+                            OpenField="@nameof(StockDataPoint.Open)"
+                            CloseField="@nameof(StockDataPoint.Close)"
+                            HighField="@nameof(StockDataPoint.High)"
+                            LowField="@nameof(StockDataPoint.Low)">
+        </StockChartSeries>
+    </StockChartSeriesItems>
+
+    <StockChartNavigator>
+        <StockChartNavigatorCategoryAxisLabels Format="" Template=""></StockChartNavigatorCategoryAxisLabels>
+        <StockChartNavigatorSeriesItems>
+            <StockChartNavigatorSeries Type="StockChartSeriesType.Line"
+                                        Name="Product 1"
+                                        Data="@StockChartProduct1Data"
+                                        Field="@(nameof(StockDataPoint.High))"
+                            CategoryField="@(nameof(StockDataPoint.Date))">
+            </StockChartNavigatorSeries>
+        </StockChartNavigatorSeriesItems>
+    </StockChartNavigator>
+
+    <StockChartLegend>
+        <StockChartLegendLabels></StockChartLegendLabels>
+    </StockChartLegend>
+
+</TelerikStockChart>
+
+@code {
+    public List<StockDataPoint> StockChartProduct1Data { get; set; }
+
+    protected override async Task OnInitializedAsync()
+    {
+        await GenerateChartData();
+    }
+
+    public async Task GenerateChartData()
+    {
+        StockChartProduct1Data = new List<StockDataPoint>()
+        {
+            new StockDataPoint(new DateTime(2019, 1, 1), 41.62m, 40.12m, 41.69m, 39.81m, 2632000),
+            new StockDataPoint(new DateTime(2019, 2, 1), 39.88m, 40.12m, 41.12m, 39.75m, 3584700),
+            new StockDataPoint(new DateTime(2019, 3, 1), 42m, 42.62m, 43.31m, 41.38m, 7631700),
+            new StockDataPoint(new DateTime(2019, 4, 1), 42.25m, 43.06m, 43.31m, 41.12m, 4922200),
+        };
+
+        await Task.FromResult(StockChartProduct1Data);
+    }
+
+    public class StockDataPoint
+    {
+        public StockDataPoint() { }
+
+        public StockDataPoint(DateTime date, decimal open, decimal close, decimal high, decimal low, int volume)
+        {
+            Date = date;
+            Open = open;
+            Close = close;
+            High = high;
+            Low = low;
+            Volume = volume;
+        }
+        public DateTime Date { get; set; }
+
+        public decimal Open { get; set; }
+
+        public decimal Close { get; set; }
+
+        public decimal High { get; set; }
+
+        public decimal Low { get; set; }
+
+        public int Volume { get; set; }
+    }
+}
+````
+
+In a **category axis label template**, you can use the following fields:
```

Comment:
Don't use bold for names or emphasis. This seems to be a proper name, so you can apply the corresponding capitalization: Category Axis Label Template